paywall/package.json

```diff
@@ -13,10 +13,10 @@
     "test": "cross-env UNLOCK_ENV=test jest --env=jsdom",
     "lint": "eslint --ext .tsx,.ts,.js src/",
     "svg-2-components": "./node_modules/@svgr/cli/bin/svgr --title-prop --no-dimensions --template src/components/interface/svg/template.js --no-dimensions -d src/components/interface/svg/ src/static/images/svg/",
-    "build-checkout": "yarn build-unlock.1.0.js && yarn build-data-iframe.1.0.js && yarn build-unlock.2.0.js",
-    "watch-checkout": "webpack --watch --config unlock.2.0.js.webpack.config.js",
+    "build-checkout": "yarn build-unlock.1.0.js && yarn build-data-iframe.1.0.js && yarn build-unlock.latest.js",
+    "watch-checkout": "webpack --watch --config unlock.latest.js.webpack.config.js",
     "build-unlock.1.0.js": "webpack --config unlock.1.0.js.webpack.config.js",
-    "build-unlock.2.0.js": "webpack --config unlock.2.0.js.webpack.config.js",
+    "build-unlock.latest.js": "webpack --config unlock.latest.js.webpack.config.js",
     "build-unlock-web": "cross-env UNLOCK_ENV=prod webpack --config unlock-web-module.webpack.config.js",
```

Comment:
```suggestion
```

We can remove this build target, it's from our previous attempt at creating an `npm` module